Assets/Scripts/Controllers/Graph/NodeController.cs

```diff
@@ -104,22 +104,26 @@ public class NodeController : MonoBehaviour {
 			GraphController = GetComponent<GraphController>();
 			NetworkController = GetComponent<NetworkController>();
 			inputController = GetComponent<InputController>();
-			NodeLoadManager = new NodeLoadManager(this);
+			NodeLoadManager = new NodeLoadManager(this);
+			NodeLoaderController = GetComponent<NodeLoaderController>();
 		}
 
 		private void Start() {
 		
 			NodeStateManager = new NodeStateManager(this);
 			Nodes.Pool = new GameObjectPool(Nodes.Prefab, Nodes.PreloadNumber, Nodes.PoolContainer);
-
-			if (NetworkController.IsServer()) {
-				for (uint i = 0; i < Math.Min(NodeLoadedLimit, NodeLoadManager.NodeLoader.GetNodeNumber()); i++)
-					NodeLoadManager.LoadNode(i);
+			if (NetworkController.IsServer()) {
+				NodeLoaderController.isReady = false;
+				for (uint i = 0; i < Math.Min(NodeLoaderController.nodeStartingAmount, NodeLoadManager.NodeLoader.GetNodeNumber()); i++) {
+					NodeLoadManager.LoadNode(i);
+					NodeLoaderController.AddLowPriorityNode(i);
+				}
```

Comment:
That's not really elegant, why not let the loader do the loading in its Start before other subscriptions?

Reply:
Fixed